### ISSUE TITLE
Lists documentation is wrong/misleading 

### ISSUE THREAD

**Issue by agvardha57** [thunkable/thunkable-issues#1219]
![image](https://user-images.githubusercontent.com/104843238/209453464-c5f29aa4-8c5d-409b-8b45-641d0f927384.png)
If you use the does list with the gear it does not work. I was told one with the gear is meant for creating a list not to search it. 
This fails:
![image](https://user-images.githubusercontent.com/104843238/209453533-5b7c56f1-362e-46fe-8b43-9e4806f4c9e0.png)
This works:
![image](https://user-images.githubusercontent.com/104843238/209453542-4df7e7be-7a78-4e97-88cf-a026501bcab5.png)
Please fix the documentation

**Comment by tatiang** [comment]
The documentation is correct but you're confusing two different things.

This:

<img width="188" alt="image" src="https://user-images.githubusercontent.com/10355498/209584786-fe055b2d-7b4a-4619-9eac-387d9b115db1.png">

Is two text strings which become a list because the [gear] list block is added to them. In this case, you need the [gear] list block.

This:

<img width="333" alt="image" src="https://user-images.githubusercontent.com/10355498/209584807-9068e78b-d5d1-4a3e-9c0b-6469c1d9bf06.png">

Is a list (`list viewer's text items` is automatically a list) within a list because the [gear] list block is added to it. So in that case, you don't need the [gear] list block.

**Comment by tatiang** [comment]
Here are some examples of blocks that are _already_ lists:

<img width="198" alt="image" src="https://user-images.githubusercontent.com/10355498/209584968-83f84946-fc65-4284-a958-e8a70ba9f26f.png">

<img width="380" alt="image" src="https://user-images.githubusercontent.com/10355498/209584982-89f75d60-b2f5-41e3-8945-73f6b956702a.png">

<img width="410" alt="image" src="https://user-images.githubusercontent.com/10355498/209585025-fc461646-ef08-40e1-a4af-8a3140e1dd07.png">

**Comment by agvardha57** [comment]
Yes got the idea Tatiang. I had read up on the Mutator blocks - https://thunkable.com/explore/ai2/support/concepts/mutators.html (The blue gear box) however every time I click on 
![image](https://user-images.githubusercontent.com/104843238/209590919-bfb7d6f8-683d-4caf-9f9c-34296b197c8e.png)
anyone below the link does not resolved just hangs. I wish they had explained it the way you explained in the original List Documentation. I will modify m bug report suitably. Thanks a lot

**Comment by tatiang** [comment]
You're welcome. By the way, that's quite outdated documentation that you're referencing. Current documentation is at https://docs.thunkable.com.
